modules/node/src/admin/admin.service.ts

```diff
@@ -52,16 +54,25 @@ export class AdminService {
   /**  Get Channels by xpub */
 
   async getChannelStates(userPublicIdentifier: any): Promise<CFCoreRecord[]> {
-    // get channel by xpub
+    // get channel by xpubgetChannelStateByMultiSig
     return await this.channelService.getChannelState(userPublicIdentifier.id);
   }
 
+  async getChannelStateByMultiSig(multisigAddress: string): Promise<CFCoreRecord[]> {
+    // get channel by xpub
+    return await this.channelService.getChannelStateByMultiSig(multisigAddress);
+  }
+
   //Get all channels
   async getAllChannelsState(): Promise<CFCoreRecord[]> {
     // get channel by xpub
     return await this.channelService.getAllChannelsState();
   }
 
+  async getAllTransfers(): Promise<any[]> {
+    return await this.transferService.fetchAllTransfers();
+  }
```

Comment:
change typing to `Transfer[]`

Reply:
^won't this break? if it's many transfers not one?